### PR TITLE
Make the popsicle appear on its stick when spawned

### PR DESCRIPTION
**Pull request by optimumtact**
🆑
fix: Popsicles will now start on their stick.
/🆑

### CODE REVIEW

File: code/game/objects/items/food/frozen.dm [hunk 1]
```diff
@@ -227,6 +227,7 @@
 /obj/item/food/popsicle/Initialize()
 	. = ..()
 	bite_consumption = reagents.total_volume / bite_states
+	update_icon() // make sure the popsical overlay is primed so it's not just a stick until you start eating it
```
Comment (Mothblocks) [inline code/game/objects/items/food/frozen.dm:230]:
```suggestion
	update_icon() // make sure the popsicle overlay is primed so it's not just a stick until you start eating it
```

Comment (maxymax13) [reply code/game/objects/items/food/frozen.dm]: soul

**Comment by Time-Green** [comment]
Don’t forget to add a changelog! 
Something like this should do:
cl: 
fix: Popsicles will now start on their stick!
/:cl:

**Comment by Time-Green** [comment]
Don’t forget to add a changelog! 
Something like this should do:
:cl: 
fix: Popsicles will now start on their stick!
/:cl:

**Comment by optimumtact** [comment]
no

**Pull request merged**